tests/models_test.py

```diff
@@ -8,25 +7,21 @@
 import larq_zoo as lqz
 
 
-def keras_test(func):
-    """Function wrapper to clean up after TensorFlow tests.
-    # Arguments
-        func: test function to clean up after.
-    # Returns
-        A function wrapping the input function.
-    """
+@pytest.fixture(autouse=True)
+def cleanup(request):
+    request.addfinalizer(keras.backend.clear_session)
 
-    @functools.wraps(func)
-    def wrapper(*args, **kwargs):
-        output = func(*args, **kwargs)
-        keras.backend.clear_session()
-        return output
 
-    return wrapper
+@pytest.fixture(scope="module")
```

Comment:
`scope="module"` will cache the result of the fixture for all tests in this file